### PR TITLE
[Feat] Proper Handler of Adding and Removal of Desktop/Menu Shortcuts

### PR DESCRIPTION
**Pull request by flavioislima**
This PR improves a few things related to shortcuts:
- [x] Add IPC handler to check if the menu or desktop shortcuts exists
- [x] Change the text Add Shortcut to Remove Shortcut if there is any
- [x] Fix Message box not showing up when adding shortcuts
- [x] Add new Message box with info on removed shortcuts

image

---

Use the following Checklist if you have changed something on the Backend or Frontend:

- [x] Tested the feature and it's working on a current and clean install.
- [x] Tested the main App features and they are still working on a current and clean install. (Login, Install, Play, Uninstall, Move games, etc.)
- [ ] Created / Updated Tests (If necessary)
- [ ] Created / Updated documentation (If necessary)

### CODE REVIEW

File: electron/main.ts [hunk 1]
```diff
@@ -681,6 +681,7 @@ ipcMain.handle('getGameInfo', async (event, appName, runner) => {
       return null
     }
     info.extra = await game.getExtraInfo()
+    // eslint-disable-next-line @typescript-eslint/return-await
```
Comment (Nocccer) [inline electron/main.ts:684]: Is this needed?

**Review by Nocccer — CHANGES_REQUESTED** [review]
LGTM. I wonder if the eslint disable is really needed? It wasn't there before and worked.

File: electron/main.ts (hunk 1 above)
Comment (flavioislima) [reply electron/main.ts:684]: yes, weird, for me it is giving an error that makes no sense.
Let me check again.

Comment (flavioislima) [reply electron/main.ts]: Yes, removed and the error is gone.
Before it was complaining that I should have an `await` there, but makes no sense and then it was complaining that the `await` was not needed.

Comment (Nocccer) [reply electron/main.ts]: I have this error sometimes if i switch branches. I think eslint or vscode is sometimes slow.
I started to use `git clean -dxf` on every branch switch.

**Review by Nocccer — APPROVED** [review]

**Pull request merged**